index.html

```diff
@@ -105,6 +107,117 @@ <h2 class="work-sans"><a id="hl-title-txt-side2" href="#">This Mac app automatic
+    				<li><a class="snc-auth work-sans" href="#">Cities Today</a></li>
+    				<li class="snc-auth work-sans">5 hours ago </li>
+    			</ul>	
+    		</div>	
+		</section>	
+
+		<section class="small-news-cont newscont4">
+    		<a class="snc-img-a" href="#"><img class="snc-img" src="img/latest-news4.jpg" alt="article pic"></a>
+
+    		<div class="work-sans snc-text">
+    			<h4><a class="snc-title" href="#">3 reasons why almost every VC investor passed on Zoom
+					</a></h4>
+    			<ul class="snc-list list-inline">
+    				<li><a class="snc-auth work-sans" href="#">Philip Levinson </a></li>
+    				<li class="snc-auth work-sans">5 hours ago</li>
+    			</ul>	
+    		</div>	
+		</section>	
+
+		<section class="small-news-cont newscont5">
+    		<a class="snc-img-a" href="#"><img class="snc-img" src="img/latest-news6.jpg" alt="article pic"></a>
+
+    		<div class="work-sans snc-text">
+    			<h4><a class="snc-title" href="#">It’s happening! World’s biggest diesel factory pivots to electric motors
+					</a></h4>
+    			<ul class="snc-list list-inline">
+    				<li><a class="snc-auth work-sans" href="#">Matthew Beedham</a></li>
+    				<li class="snc-auth work-sans">6 hours ago </li>
+    			</ul>	
+    		</div>	
+		</section>	
+
+		<section class="small-news-cont newscont6">
+    		<a class="snc-img-a" href="#"><img class="snc-img" src="img/latest-news6.jpg" alt="article pic"></a>
+
+    		<div class="work-sans snc-text">
+    			<h4><a class="snc-title" href="#">This nearly inexplicable exoplanet has the density of cotton candy
+					</a></h4>
+    			<ul class="snc-list list-inline">
+    				<li><a class="snc-auth work-sans" href="#">The Cosmic Companion</a></li>
+    				<li class="snc-auth work-sans">6 hours ago </li>
+    			</ul>	
+    		</div>	
+		</section>	
+
+		<section id="last-two" class="small-news-cont newscont7">
+    		<a class="snc-img-a" href="#"><img class="snc-img" src="img/latest-news7.jpg" alt="article pic"></a>
+
+    		<div class="work-sans snc-text">
+    			<h4><a class="snc-title" href="#">The future of US government vehicles is electric
+					</a></h4>
+    			<ul class="snc-list list-inline">
+    				<li><a class="snc-auth work-sans" href="#">Matthew Beedham  </a></li>
+    				<li class="snc-auth work-sans">7 hours ago</li>
+    			</ul>	
+    		</div>	
+		</section>	
+
+		<section class="small-news-cont newscont8">
+    		<a class="snc-img-a" href="#"><img class="snc-img" src="img/latest-news8.png" alt="article pic"></a>
+
+    		<div class="work-sans snc-text">
+    			<h4><a class="snc-title" href="#">Signal is drama-free for now, but it should prepare for the worst
+					</a></h4>
+    			<ul class="snc-list list-inline">
+    				<li><a class="snc-auth work-sans" href="#">Abhimanyu Ghoshal </a></li>
+    				<li class="snc-auth work-sans">8 hours ago</li>
+    			</ul>	
+    		</div>	
+		</section>
+
+		<section id="small-news-last">
+
+		</section>		
+
+    </section>	
+
```

Comment:
Articles in this section should use the article tag, the latest section should be a wrapper generally, same for other sections.